docroot/themes/humsci/humsci_basic/readme.md

```diff
@@ -0,0 +1,68 @@
+# Humsci Basic (humsci_basic) Theme
+
+Humsci Basic (humsci_basic) is a theme that can be used as a base for all future H&S themes. We have pulled patterns and templates for Humsci Basic from Stanford Basic (stanford_basic) and Stanford University Humsci Theme (su_humsci_theme) Themes. Humsci Basic integrates selectively with Decanter Sass, styles, and patterns.
```

Comment:
I'd remove the "all" here, since I could imagine us continuing to use Stanford Basic / Humsci Theme for some future site-specific themes, if it makes sense.